sycl/include/sycl/detail/pi.h

```diff
@@ -356,6 +356,7 @@ typedef enum {
   PI_EXT_INTEL_DEVICE_INFO_MEM_CHANNEL_SUPPORT = 0x20008,
   // The number of max registers per block (device specific)
   PI_EXT_CODEPLAY_DEVICE_INFO_MAX_REGISTERS_PER_WORK_GROUP = 0x20009,
+  PI_EXT_ONEAPI_DEVICE_INFO_IP_VERSION = 0x4250,
```

Comment:
nit: any reason why this is not consecutive after PI_EXT_CODEPLAY_DEVICE_INFO_MAX_REGISTERS_PER_WORK_GROUP?

Reply:
Answered in https://github.com/intel/llvm/pull/9843#discussion_r1228241355 why it is `0x4250`, but this line may be moved to another place to avoid the confusion.

Reply:
Moved above